assertj-core/src/main/java/org/assertj/core/api/recursive/comparison/DualValue.java

```diff
@@ -322,7 +322,7 @@ public boolean hasPotentialCyclingValues() {
 
   private static boolean isAJsonValueNode(Object value) {
     try {
-      Class<?> valueNodeClass = Class.forName("com.fasterxml.jackson.databind.node.ValueNode");
+      Class<?> valueNodeClass = Class.forName("tools.jackson.databind.node.ValueNode");
```

Comment:
Here we're practically dropping support for Jackson 2.

What do you think about having support for both versions/namespaces, along with their corresponding test cases?